docs/modules/deployment/pages/core/docker/core.adoc

```diff
@@ -35,27 +31,31 @@ services:
       OPENNMS_DBPASS: 'my-opennms-db-password'
     volumes:<6>
       - data-opennms:/opennms-data
-      - ./etc:/opt/opennms/etc<7>
+      - data-config:/opt/opennms/etc
     command: ["-s"]
-    ports:<8>
+    ports:<7>
       - '8980:8980/tcp'
       - '8101:8101/tcp'
     healthcheck:
-      test: [ 'CMD', 'curl', '-f', '-I', 'http://localhost:8980/opennms/login.jsp' ]<9>
+      test: [ 'CMD', 'curl', '-f', '-I', 'http://localhost:8980/opennms/login.jsp' ]<8>
       interval: 1m
       timeout: 5s
       retries: 3
 ----
 
-<1> Volume definitions to persist the {page-component-title} Core data; for example, RRD files and PDF reports.
+<1> Volume definitions to persist the {page-component-title} Core data and configuration files.
 <2> The {page-component-title} Core instance service is named `horizon` with a friendly `container_name`.
 <3> Image reference using the {page-component-title} container image with the Core services.
 <4> Set the time zone and the postgres credentials to initialize the database that the {page-component-title} Core instance uses. To list all available time zones, use `timedatectl list-timezones`.
 <5> Set the host or IP address of the host that runs the PostgreSQL database.
-<6> Persist performance data PDF reports in a volume.
-<7> Mount the volumes for data persistence and bind mount the configuration in a local directory.
-<8> Publish ports to access the web UI and the Karaf shell.
-<9> Run an internal health check against the web UI to verify service health status.
+<6> Mount directories to store RRD files, PDF reports, and configuration files in a persistent volume.
+<7> Publish ports to access the web UI and the Karaf shell.
+<8> Run an internal health check against the web UI to verify service health status.
+
+TIP: The process inside the container runs as a non-privileged user with user id `10001`.
```

Comment:
```suggestion
NOTE: The process inside the container runs as a non-privileged user with user id `10001`.
```

Reply:
@indigo423 could you make this change a merge this PR?